docs/source/pythonapi/openmoc_compatible.rst

```diff
@@ -0,0 +1,8 @@
+.. _pythonapi_openmoc_compatible:
+
+====================
+OpenMOC Compatibility
+====================
```

Comment:
Make sure = covers the title (need one extra above and below). Not being nitpicky -- this is actually a rst rule